Apps/Examples/Examples/Models/ExampleProtocol.swift

```diff
@@ -8,16 +8,7 @@ public protocol ExampleProtocol: AnyObject {
 
 extension ExampleProtocol {
     public func resourceOptions() -> ResourceOptions {
-        guard let accessToken = AccountManager.shared.accessToken else {
-            fatalError("Access token not set")
-        }
-
-        guard !accessToken.isEmpty else {
-            fatalError("Empty access token")
-        }
-
-        let resourceOptions = ResourceOptions(accessToken: accessToken)
-        return resourceOptions
+        return ResourceOptions(accessToken: CredentialsManager.default.accessToken)
```

Comment:
in Kotlin we can not extend the constructor of the ResourceOptions(generated final class), I'm thinking shall we expose a `CredentialsManager#getResourceOptions()` API in CredentialsManager? cc @Chaoba

Reply:
@pengdev Yes, that would be good. We would need to create a MapInitOption instance for offline

Reply:
I'll update to do the same, so for iOS it would be (for example) `CredentialsManager.default.resourceOptions`. Developers would still be able to use the initializer way too.